### DISCUSSION TITLE
Simple example of add a list item with person field

### DISCUSSION THREAD

**Discussion by Ofer-Gal** [pnp/pnpcore#322] [Q&A]
The example in the documentation https://pnp.github.io/pnpcore/using-the-sdk/listitems-fields.html#user-fields 
Adds the "MyField" column to the list and then uses it to add a ListItem.
What about When the person column already exists? 
I tried to create a "FieldUserValue" and add the user created with EnsureUserAsync but it returns a REST error.
Why doesn't  `values.Add("ManagerLookupId", myUser.Id );` or `values.Add("ManagerId", myUser.Id );`  work ?

Where can I find simple example?

I managed with 
```
 IField managerField = myList.Fields.First(f=> f.InternalName == "Manager");
 values.Add("Manager" , managerField.NewFieldUserValue(myUser));

```
Is this the simplest way to do it? seems too much 😒🤦‍♂️

**Comment by jansenbe** [comment]
To correctly handle the "complex" fields we indeed need to have the field information loaded, your sample is OK. https://pnp.github.io/pnpcore/using-the-sdk/listitems-intro.html#reading-list-items describes how to be best ensure all the fields of the list are installed.
